tests/framework/test_time.py

```diff
@@ -20,52 +29,44 @@ def components():
     ]
 
 
+class StepModifier(MockGenericComponent):
+    def __init__(self, name, step_modifier):
+        super().__init__(name)
+        self.step_modifier = step_modifier
+
+    def setup(self, builder) -> None:
+        super().setup(builder)
+        builder.value.register_value_modifier("simulant_step_size", self.modify_step)
+
+    def modify_step(self, index):
+        return pd.Series(pd.Timedelta(days=self.step_modifier), index=index)
+
+
 def test_align_times(SimulationContext, base_config, components):
     sim = SimulationContext(base_config, components)
     sim.setup()
     sim.initialize_simulants()
     pop_size = len(sim.get_population())
-    # After initialization, all simulants should be aligned to event times
-    assert (
-        len(
-            sim._clock.get_active_population(
-                sim.get_population().index,
-                sim._clock.event_time,
-            )
-        )
-        == pop_size
+    active_simulants = lambda: sim._clock.get_active_population(
+        sim.get_population().index,
+        sim._clock.event_time,
     )
```

Comment:
Since you're calling this method in multiple tests, you should probably bring it outside the test:
```python
def active_simulants(sim):
    return sim._clock.get_active_population(
        sim.get_population().index, sim._clock.event_time
    )
```